# HG changeset patch
# User Stelios Voutsinas <stv@roe.ac.uk>
# Date 1417622745 -7200
#      Wed Dec 03 18:05:45 2014 +0200
# Branch 1.11.28-stv-cds-adql-2
# Node ID ff3ba2bc9b0e9f33d72bfa8534c827da14e9c916
# Parent  d9d9f7503165aa87eac82bd159dd70cccbecec78
Remove final from some CDS classes where needed for override [CDS-Patch]

diff -r d9d9f7503165 -r ff3ba2bc9b0e adql/db/DBChecker.java
--- a/adql/db/DBChecker.java	Wed Dec 03 18:03:49 2014 +0200
+++ b/adql/db/DBChecker.java	Wed Dec 03 18:05:45 2014 +0200
@@ -364,7 +364,7 @@
 	 * @see #check(ADQLQuery, Stack)
 	 */
 	@Override
-	public final void check(final ADQLQuery query) throws ParseException{
+	public void check(final ADQLQuery query) throws ParseException{
 		check(query, null);
 	}
 
diff -r d9d9f7503165 -r ff3ba2bc9b0e adql/translator/JDBCTranslator.java
--- a/adql/translator/JDBCTranslator.java	Wed Dec 03 18:03:49 2014 +0200
+++ b/adql/translator/JDBCTranslator.java	Wed Dec 03 18:05:45 2014 +0200
@@ -742,7 +742,7 @@
 	 * 
 	 * @throws TranslationException	If there is an error during the translation.
 	 */
-	protected final String getDefaultADQLFunction(ADQLFunction fct) throws TranslationException{
+	protected String getDefaultADQLFunction(ADQLFunction fct) throws TranslationException{
 		String sql = fct.getName() + "(";
 
 		for(int i = 0; i < fct.getNbParameters(); i++)
